### ISSUE TITLE
what's matter

### ISSUE THREAD

**Issue by wyjzz** [Microsoft/CNTK#282]
tarting minibatch loop.
RandomOrdering: 703 retries for 3566 elements (19.7%) to ensure window condition
RandomOrdering: recached sequence for seed 1: 1485, 1417, ...
 Epoch[ 2 of 4]-Minibatch[   1-  10, 0.000000000000003%]: SamplesSeen = 160; TrainLossPerSample =  0.51669922; EvalErr[0]PerSample = 0.21250000; TotalTime = 1.1168s; SamplesPerSecond = 143.3
 Epoch[ 2 of 4]-Minibatch[  11-  20, 0.000000000000007%]: SamplesSeen = 160; TrainLossPerSample =  0.61068220; EvalErr[0]PerSample = 0.19375000; TotalTime = 1.0686s; SamplesPerSecond = 149.7
 Epoch[ 2 of 4]-Minibatch[  21-  30, 0.000000000000010%]: SamplesSeen = 160; TrainLossPerSample =  0.72498369; EvalErr[0]PerSample = 0.65625000; TotalTime = 1.0657s; SamplesPerSecond = 150.1
 Epoch[ 2 of 4]-Minibatch[  31-  40, 0.000000000000014%]: SamplesSeen = 160; TrainLossPerSample =  0.71991577; EvalErr[0]PerSample = 0.61250000; TotalTime = 1.0668s; SamplesPerSecond = 150.0
 Epoch[ 2 of 4]-Minibatch[  41-  50, 0.000000000000017%]: SamplesSeen = 160; TrainLossPerSample =  0.63410378; EvalErr[0]PerSample = 0.20625000; TotalTime = 1.0844s; SamplesPerSecond = 147.6
 Epoch[ 2 of 4]-Minibatch[  51-  60, 0.000000000000021%]: SamplesSeen = 160; TrainLossPerSample =  0.57162132; EvalErr[0]PerSample = 0.19375000; TotalTime = 1.0543s; SamplesPerSecond = 151.8
 Epoch[ 2 of 4]-Minibatch[  61-  70, 0.000000000000024%]: SamplesSeen = 160; TrainLossPerSample =  0.56749840; EvalErr[0]PerSample = 0.20000000; TotalTime = 1.0630s; SamplesPerSecond = 150.5
 Epoch[ 2 of 4]-Minibatch[  71-  80, 0.000000000000028%]: SamplesSeen = 160; TrainLossPerSample =  0.56872520; EvalErr[0]PerSample = 0.18750000; TotalTime = 1.0737s; SamplesPerSecond = 149.0
 Epoch[ 2 of 4]-Minibatch[  81-  90, 0.000000000000031%]: SamplesSeen = 160; TrainLossPerSample =  0.54473305; EvalErr[0]PerSample = 0.16875000; TotalTime = 1.1331s; SamplesPerSecond = 141.2
 Epoch[ 2 of 4]-Minibatch[  91- 100, 0.000000000000035%]: SamplesSeen = 160; TrainLossPerSample =  0.50428276; EvalErr[0]PerSample = 0.15000000; TotalTime = 1.1455s; SamplesPerSecond = 139.7
 Epoch[ 2 of 4]-Minibatch[ 101- 110, 0.000000000000038%]: SamplesSeen = 160; TrainLossPerSample =  0.74079056; EvalErr[0]PerSample = 0.23750000; TotalTime = 1.0847s; SamplesPerSecond = 147.5
 Epoch[ 2 of 4]-Minibatch[ 111- 120, 0.000000000000042%]: SamplesSeen = 160; TrainLossPerSample =  0.55250702; EvalErr[0]PerSample = 0.16875000; TotalTime = 1.1470s; SamplesPerSecond = 139.5
 Epoch[ 2 of 4]-Minibatch[ 121- 130, 0.000000000000045%]: SamplesSeen = 160; TrainLossPerSample =  0.50932846; EvalErr[0]PerSample = 0.15625000; TotalTime = 1.5292s; SamplesPerSecond = 104.6
 Epoch[ 2 of 4]-Minibatch[ 131- 140, 0.000000000000049%]: SamplesSeen = 160; TrainLossPerSample =  0.59688873; EvalErr[0]PerSample = 0.19375000; TotalTime = 1.1665s; SamplesPerSecond = 137.2
 Epoch[ 2 of 4]-Minibatch[ 141- 150, 0.000000000000052%]: SamplesSeen = 160; TrainLossPerSample =  0.57491226; EvalErr[0]PerSample = 0.19375000; TotalTime = 1.0667s; SamplesPerSecond = 150.0
 Epoch[ 2 of 4]-Minibatch[ 151- 160, 0.000000000000056%]: SamplesSeen = 160; TrainLossPerSample =  0.44013519; EvalErr[0]PerSample = 0.14375000; TotalTime = 1.0834s; SamplesPerSecond = 147.7
 Epoch[ 2 of 4]-Minibatch[ 161- 170, 0.000000000000059%]: SamplesSeen = 160; TrainLossPerSample =  0.49790268; EvalErr[0]PerSample = 0.17500000; TotalTime = 1.0649s; SamplesPerSecond = 150.2
 Epoch[ 2 of 4]-Minibatch[ 171- 180, 0.000000000000062%]: SamplesSeen = 160; TrainLossPerSample =  0.43228226; EvalErr[0]PerSample = 0.15000000; TotalTime = 1.1395s; SamplesPerSecond = 140.4
 Epoch[ 2 of 4]-Minibatch[ 181- 190, 0.000000000000066%]: SamplesSeen = 160; TrainLossPerSample =  0.45122757; EvalErr[0]PerSample = 0.16250000; TotalTime = 1.1413s; SamplesPerSecond = 140.2
 Epoch[ 2 of 4]-Minibatch[ 191- 200, 0.000000000000069%]: SamplesSeen = 160; TrainLossPerSample =  0.53693085; EvalErr[0]PerSample = 0.21875000; TotalTime = 1.0754s; SamplesPerSecond = 148.8
 Epoch[ 2 of 4]-Minibatch[ 201- 210, 0.000000000000073%]: SamplesSeen = 160; TrainLossPerSample =  0.40316849; EvalErr[0]PerSample = 0.11250000; TotalTime = 1.0608s; SamplesPerSecond = 150.8
 Epoch[ 2 of 4]-Minibatch[ 211- 220, 0.000000000000076%]: SamplesSeen = 160; TrainLossPerSample =  0.46430969; EvalErr[0]PerSample = 0.15625000; TotalTime = 1.0800s; SamplesPerSecond = 148.1
Finished Epoch[ 2 of 4]: [Training Set] TrainLossPerSample = 0.55231541; EvalErrPerSample = 0.22013462; AvgLearningRatePerSample = 0.03125; EpochTime=25.4034
Starting Epoch 3: learning rate per sample = 0.002232  effective momentum = 0.900000  momentum as time constant = 2126.0 samples

[CALL STACK]
    >00007FFD199CF0B3 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -Microsoft::MSR::CNTK::GetReader<double>
    -Microsoft::MSR::CNTK::GetReader<double>
    -00007FF68CD50A03 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CE9C4B8 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CE98E5D (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CE96A3E (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CE8B27E (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CDBAE8D (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CD832E6 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CD83768 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CD83812 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -00007FF68CF21F63 (SymFromAddr error : 璇曞浘璁块棶鏃犳晥鐨勫湴鍧€銆?
    -BaseThreadInitThunk
    -RtlUserThreadStart

EXCEPTION occurred: randomizeRange must be an even multiple of mbSize and an integral factor of epochSize

**Comment by 1132520084** [comment]
You would get some useful information in the [issues/255](https://github.com/Microsoft/CNTK/issues/255)
